### PR TITLE
LPS-137112

### PR DESCRIPTION
**Pull request by mbowerman**
I modified this according to what @brianchandotcom asked for in https://github.com/brianchandotcom/liferay-portal/pull/105777#issuecomment-901839743.

I've tested this on Oracle and MySQL. It worked on both.

https://issues.liferay.com/browse/LPS-137112

### CODE REVIEW

File: portal-impl/src/custom-sql/documentlibrary.xml [hunk 1]
```diff
@@ -138,7 +138,7 @@
 	<sql id="com.liferay.document.library.kernel.service.persistence.DLFileEntryFinder.findByExtraSettings">
 		<![CDATA[
 			SELECT
-				DISTINCT {DLFileEntry.*}
+				DISTINCT DLFileEntry.fileEntryId, DLFileEntry.modifiedDate
```
Comment (adolfopa) [inline portal-impl/src/custom-sql/documentlibrary.xml:141]: Why are we including the modified date? Is it necessary for the order by clause?

Comment (mbowerman) [reply portal-impl/src/custom-sql/documentlibrary.xml:141]: Yes, if we don't include modifiedDate, then the query will error out due to the modifiedDate being used in the order by clause. In Oracle, the error is "ORA-01791: not a SELECTed expression".

**Comment by adolfopa** [comment]
ci:forward

**Review by adolfopa — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by mbowerman** [comment]
ci:test:relevant

**Comment by mbowerman** [comment]
ci:test:stable

**Comment by mbowerman** [comment]
ci:forward

**Comment by mbowerman** [comment]
@AliciaGarciaGarcia I hope you don't mind, I ran `ci:forward` since I saw you had previously run it, but when you ran it it did not get fowarded because the tests failed (I think it was some flaky or recently broken tests, not related to this fix).

Thanks for the review!

**Pull request closed without merging**